src/flb_log.c

```diff
@@ -561,20 +561,46 @@ struct flb_log *flb_log_create(struct flb_config *config, int type,
     return log;
 }
 
+#ifdef _WIN32
```

Comment:
I suggest to use the CFL API where we already have a portable way to get the current time:

https://github.com/fluent/fluent-bit/blob/master/lib/cfl/src/cfl_time.c#L31